beta/src/content/community/meetups.md

```diff
@@ -95,8 +95,8 @@ Do you have a local React.js meetup? Add it here! (Please keep the list alphabet
 * [Budapest](https://www.meetup.com/React-Budapest/)
 
 ## India {/*india*/}
-* [Bangalore](https://www.meetup.com/ReactJS-Bangalore/)
-* [Bangalore](https://www.meetup.com/React-Native-Bangalore-Meetup)
+* [Bengaluru](https://www.meetup.com/ReactJS-Bangalore/)
+* [Bengaluru (React Native)](https://www.meetup.com/React-Native-Bangalore-Meetup)
```

Comment:
we already have some places where React Native is mentioned like `[Charlotte, NC - React Native](https://www.meetup.com/cltreactnative/)` Can we do similar thing? 
```suggestion
* [Bengaluru - React Native](https://www.meetup.com/React-Native-Bangalore-Meetup)
```